ui/txs/TxsTableItem.tsx

```diff
@@ -180,7 +197,29 @@ const TxsTableItem = ({
       </Td>
       { !config.UI.views.tx.hiddenFields?.value && (
         <Td isNumeric>
-          <CurrencyValue value={ tx.value } accuracy={ 8 } isLoading={ isLoading }/>
+          { (dayjs().diff(dayjs(tx.timestamp), 'day') < 1 || Number(tx.value) === 0) ?
+            (
+              <CurrencyValue
```

Comment:
Remove duplicate code. The CurrencyValue component appears twice with identical props